otx/algorithms/segmentation/adapters/mmseg/configurer.py

```diff
@@ -501,10 +502,13 @@ def configure_task(self, cfg: ConfigDict, training: bool) -> None:
         """Patch config to support incremental learning."""
         super().configure_task(cfg, training)
 
-        new_classes: List[str] = np.setdiff1d(self.model_classes, self.org_model_classes).tolist()
-
-        # Check if new classes are added
-        has_new_class: bool = len(new_classes) > 0
+        # TODO: Revisit this part when removing bg label -> it should be 1 because of 'background' label
+        if len(set(self.org_model_classes) & set(self.model_classes)) == 1 or set(self.org_model_classes) == set(
```

Comment:
Shouldn't `has_new_class` be True also in case where only common class is a background?

Reply:
In configure_classes, bg class is always inserted in original ckpt model class. So in default, new model class(data class) and original model class always have a intersection of 'background' class. Additionally, in the case of [bg, A] => [bg, B], I don't deem 'B' as new class, because if has_new_class becomes true, it set to use class incremental sampler- but no class incremental case. I think `has_new_class` is somewhat improper name here, I'd change it to `is_cls_incr`. How is it?

Reply:
Yes I agree. but previous code consider [bg, A] => [bg, B] case as incremental case. So, this changes the policy to decide which case is incremental. Is it ok? If yes, then I'm ok too

Reply:
Yes, I intended. Only class is newly added compared to previous, it has to use our custom cls_incr sampler.